packages/storybook-utils/src/preview.ts

```diff
@@ -134,6 +119,29 @@ export const createPreview = <T extends Preview = Preview>(overrides?: T) => {
   return deepmerge<[T, typeof defaultPreview]>(overrides ?? ({} as T), defaultPreview);
 };
 
+/**
+ * Custom transformer for the story source code to better fit to our
+ * Vue.js code because storybook per default does not render it exactly how
+ * we want it to look.
+ * @see https://storybook.js.org/docs/react/api/doc-block-source
+ */
+export const sourceCodeTransformer = (sourceCode: string): string => {
+  const replacements = [
+    // replace event bindings with shortcut
+    { searchValue: "v-on:", replaceValue: "@" },
+    // remove empty event handlers, e.g. @click="()=>({})" will be removed
+    { searchValue: / @.*['"]\(\)=>\({}\)['"]/g, replaceValue: "" },
+    // remove empty v-binds, e.g. v-bind="{}" will be removed
+    { searchValue: / v-bind=['"]{}['"]/g, replaceValue: "" },
+    // replace boolean shortcuts for true, e.g. disabled="true" will be changed to just disabled
+    { searchValue: /:(.*)=['"]true['"]/g, replaceValue: "$1" },
+  ];
+
+  return replacements.reduce((code, replacement) => {
+    return replaceAll(code, replacement.searchValue, replacement.replaceValue);
+  }, sourceCode);
+};
```

Comment:
Regex are tricky, would be nice to have a test for this function

Reply:
done